chain/client/src/info.rs

```diff
@@ -144,10 +145,11 @@ impl InfoHelper {
 
         let validator_info_log = validator_info.as_ref().map(|info| {
             format!(
-                " {}{} validator{}",
+                " {}{} validator{} (of which {} are chunk only)",
```

Comment:
let's get the naming right:

we have 3 tasks in our blockchain:
- block producer (creates a block)
- chunk producer (creates a chunk)
- validator (checks that blocks/chunks are valid and add its signature).

And what we do is:
- top 100 nodes have all 3 tasks
- another (up to 300 nodes) have only chunk production.

So technically we should write things in this log like:
"100 validators (+ 10 chunk-only producers)"

And also for the first entry here ('is_validator") - it should also distinguish whether your node is a validator or chunk-only producer.

Reply:
I corrected the nomenclature used, but I'm not sure how to determine if a node is a chunk only producer.